src/snovault/schema_utils.py

```diff
@@ -250,6 +250,17 @@ def permission(validator, permission, instance, schema):
         yield IgnoreUnchanged(error)
 
 
+orig_uniqueItems = Draft4Validator.VALIDATORS['uniqueItems']
+
+
+def uniqueItems(validator, uI, instance, schema):
+    # Use serialized items if available
+    # (this gives the linkTo validator a chance to normalize paths into uuids)
+    if validator._serialize and validator._validated[-1]:
+        instance = validator._validated[-1]
+    yield from orig_uniqueItems(validator, uI, instance, schema)
```

Comment:
Why use 'from'?  I haven't read much about 'yield from' but it seems like yield would be fine.

In the BaseValidator code,
https://github.com/lrowe/jsonschema_serialize_fork/blob/master/jsonschema_serialize_fork/validators.py 
the iter_errors gets orig_uniqueItems as a generator on line 110.

Then it's looped over and yields the errors.  It's base code, so we can't change it, but shouldn't the 'yield from' be on 121? 

Now having looked into it, I wonder if we need a yield at all in the uniqueItems wrapper?  Should we yield the orig_uniqueItems generator function or just return it directly?

Reply:
`yield from` lets a generator to delegate to another generator (see PEP 380). So this line is equivalent to:

```python
for error in orig_uniqueItems(validator, uI, instance, schema):
    yield error
```

I hadn't looked at how validators are executed, so was assuming it is important for the `uniqueItems` validator to continue to be a generator. As it turns out, it looks like it would probably be okay to just do `return orig_uniqueItems(validator, uI, instance, schema)` since `iter_errors` only assumes the return value of a validator is iterable, not that it is necessarily a generator. I'd still feel more comfortable keeping it as a generator since I haven't done a thorough hunt to see if there are other call sites.